README.md

```diff
@@ -45,22 +45,84 @@ Modern browsers do not provide APIs to directly use TCP or UDP. WebVM provides n
 
 You can now customize `dockerfiles/debian_mini` to suit your needs, or make a new Dockerfile from scratch. Use the `Path to Dockerfile` workflow parameter to select it.
 
-# Local deployment
-
-From a local `git clone`
-
-- Download the `debian_mini` Ext2 image from [https://github.com/leaningtech/webvm/releases/](https://github.com/leaningtech/webvm/releases/)
-	- You can also build your own by selecting the "Upload GitHub release" workflow option
-	- Place the image in the repository root folder
-- Edit `config_github_terminal.js`
-	- Uncomment the default values for `CMD`, `ARGS`, `ENV` and `CWD`
-	- Replace `IMAGE_URL` with the URL (absolute or relative) for the Ext2 image. For example `"/debian_mini_20230519_5022088024.ext2"`
-- Build WebVM using `npm`, output will be placed in the `build` directory
-	- `npm install`
-	- `npm run build`
-- Start NGINX, it automatically points to the `build` directory just created
-	- `nginx -p . -c nginx.conf`
-- Visit `http://127.0.0.1:8081` and enjoy your local WebVM
+# Run WebVM locally with a custom Debian mini disk image
+
+1. Clone the WebVM Repository
+
+```sh
+git clone https://github.com/leaningtech/webvm.git
+cd webvm
+```
+
+2. Download the Debian mini Ext2 image
+	- Download the `debian_mini` Ext2 image from [https://github.com/leaningtech/webvm/releases/](https://github.com/leaningtech/webvm/releases/)
```

Comment:
We can do better here, let's provide a one liner `wget` / `curl` command to get the release image. Mentioning that people can generated their own make sense of course.